terraform/modules/tf_stream_alert/iam.tf

```diff
@@ -104,6 +104,16 @@ resource "aws_iam_role_policy" "streamalert_alert_processor_s3" {
       "Resource": [
         "${aws_s3_bucket.streamalerts.arn}/*"
       ]
+    },
+    {
+      "Action": [
+        "s3:GetObject",
+        "s3:ListBucket"
```

Comment:
I don't think you actually need `s3:ListBucket`. Unless you're literally paginating over all the items in the bucket, `GetObject` should suffice